features/deepl.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Deepl module
```

Comment:
We need to pay for those requests?

Reply:
Yes, 0,01 Euro for 500 characters and in test there is only word with 3 characters. I was aware of this and I thought it is not a problem, What do you think about that? We should comment Deepl tests?

Reply:
Yes, we should :) We need to find mock deepl client